FinSearchUnified/Tests/Helper/StaticHelperTest.php

```diff
@@ -630,7 +630,7 @@ public function testBuildCategoryName($categoryId, $category, $expected)
     private function updateParentCategoryName(Category $parent, $restore = true)
     {
         // Stop when Shopware's root category is reached. Changing it can and will break unrelated tests.
-        if ($parent->getId() === 1) {
+        if ($parent->getId() <= 3) {
```

Comment:
Why was this change necessary?

Reply:
@TheKeymaster I was trying to figure out something as whenever I execute the tests, I get like loads of shop entries created in the s_core_shops table and some "Deutsch" categories in the s_categories table. This over the time makes the tests really really slow as there are a lot of shop entries it has to go through. 
Can you please verify it at your end as well. I had discussed this with Ismail as well but he could not figure out it either why this was happening. Maybe you would be able to identify something related to it.

Reply:
@zaifastafa when are these entries created? Are they created when you execute a specific test case? Please investigate this further.

Reply:
@TheKeymaster am not able to figure out which test causes this problem. I know its some test in StaticHelperTest, and it only happens when all tests are executed together. But its not guaranteed that these entries are created everytime. It does not seem to have any pattern AFAIK. Sometimes it creates these entries, sometimes it doesn't which makes it difficult to identify the root. 

I had mentioned this to Ismail as well but he was not able to identify the problem either.

Reply:
I'd suggest changing it locally when it happens, instead of changing it for everyone, because I also haven't experienced such an issue when running tests. Please revert this change and add it if you need it locally, to improve the performance of your tests.